### ISSUE TITLE
'Add site whitelist' should be a button instead of a link ⛓

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#259]
Semantically it doesn't link to anywhere so it should be a button.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43591971-8da6879a-9642-11e8-89a4-2c358bd03c9b.png">
